docs/api/graphql/operation.mdx

```diff
@@ -55,12 +55,16 @@ const root = {
 graphql.operation(async (req, res, ctx) => {
   const payload = await graphqlRequest(
     schema,
-    req.body,
+    req.body.query,
     root,
     null,
-    req.variables,
+    req.body.variables,
   )
 
-  return res(payload)
+  if (payload.errors) {
+    return res(ctx.errors(payload.errors))
+  }
+
+  return res(ctx.data(payload))
```

Comment:
If `payload` represents the entire operation payload, then we should pass `payload.data` to the `ctx.data` utility.